src/view.c

```diff
@@ -164,7 +172,24 @@ scroll_view(struct view *view, enum request request)
 		die("request %d not handled in switch", request);
 	}
 
+	unsigned long orig_offset = view->pos.offset;
```

Comment:
I think the style convention here is to always declare variables at the beginning of a block (pre-C99).